docs/slurm/sbatch.md

```diff
@@ -826,37 +839,30 @@ mpirun --cpu-set 8-11 --bind-to core -np 4 ./your-mpi-app2 &
 wait
 ```
 
-Here, the first `mpirun` uses 8 CPU cores for eight tasks, and the second uses another 4 CPU cores to avoid oversubscription. The two "&" mean to launch the `mpirun` commands to the background, and the wait command ensures all the processes are complete before terminating the job.
-
-### Cron-like jobs - Midway2 - Legacy 
+Here, the first `mpirun` uses 8 CPU cores for eight tasks, and the second uses another 4 CPU cores to avoid oversubscription. The two "&" mean to launch the `mpirun` commands to the background, and the `wait` command ensures all the processes are complete before terminating the job.
 
-Cron-like jobs are jobs submitted to the queue with a specified schedule. These jobs persist until they are canceled or encounter an error. The Midway2 cluster has a dedicated partition, `cron`, for running cron-like jobs. Please contact our [Help Desk](https://rcc.uchicago.edu/support-and-services/consulting-and-technical-support){:target='_blank'} to request submitting Cron-like jobs. These jobs are subject to scheduling limits and will be monitored. We strongly recommend using `dependency jobs` rather than `cron jobs` since cron-like jobs is a legacy service and we are moving away from it. 
-
-Here is an example of a batch script that internally submits a Cron job (`cron.sbatch`):
-
-```bash
-#!/bin/bash
+A common use case is to launch multiple Python instances with the same python script, each processing a set of input paramters:
 
-#SBATCH --time=00:05:00
-#SBATCH --output=cron.log
-#SBATCH --open-mode=append
-#SBATCH --account=cron-account
-#SBATCH --partition=cron
-#SBATCH --qos=cron
+```
+#!/bin/sh
```

Comment:
should we use bash on all of our examples instead of sh?

Reply:
Fixed, all to bash